api/sample.env

```diff
@@ -620,3 +619,31 @@ TEST_LOG_ENABLED=false
 # type: Url
 # default: none
 TEST_REDIS_URL=redis://localhost:6379
+
+# ========
+# RATE LIMITING
+# ========
+
+# Enable rate limiting
+# presence: optional
+# type: boolean
+# default: false
+RATE_LIMIT_ENABLED=false
+
+# Log over limit error but don't block them
+# presence: optional
+# type: boolean
+# default: true
+RATE_LIMIT_LOG_ONLY=true
+
+# Default number of request allowed withing window
+# presence: optional
+# type: number
+# default: 10
+RATE_LIMIT_DEFAULT_LIMIT=10
```

Comment:
**question**
C'est quoi exactement ?
la doc est pas super claire, peut être mentionner un exemple ?
https://github.com/lob/hapi-rate-limiter#defaultrate

Reply:
: # of max requests allows within window (integer)

Reply:
Déso je voulais parler de `RATE_LIMIT_DEFAULT_WINDOW`